raft.c

```diff
@@ -1339,6 +1411,91 @@ void RaftReqHandleQueue(uv_async_t *handle)
  * Implementation of specific request types.
  */
 
+static void handleTransferLeaderComplete(raft_server_t *raft, raft_transfer_state_e state)
+{
+    raft_node_id_t transfer_id = raft_get_transfer_leader(raft);
+    raft_node_id_t leader = raft_get_leader_id(raft);
+
+    if (!redis_raft.transfer_req) {
+        LOG_ERROR("leader transfer update: but no req to correlate it to!");
+        return;
+    }
+
+    if (!transfer_id) {
```

Comment:
This will be RAFT_NODE_ID_NODE, not zero and check will not work. A bit more than that, I think we wouldn't call this callback if there is no transfer leader operation. So, we should be okay even without this check, maybe delete it?

Reply:
ok, done